src/components/DrawingShapes.jsx

```diff
@@ -0,0 +1,125 @@
+import { useEffect, useState } from "react";
+import "../style/drawShapes.css";
```

Comment:
Can you please remove this line ?

You can't import a file which not exists, Due to that Vercel is failing to deploying the application.

![Screenshot 2024-05-18 at 22-45-53 Dashboard](https://github.com/singodiyashubham87/Draw-it-out/assets/92252895/d382bd1e-902d-45f3-9714-09ff95bef9b4)

Reply:
thank you :) 
done, if u can remove circle.png also